ios/RNFetchBlob/RNFetchBlob.m

```diff
@@ -372,7 +372,7 @@ - (NSDictionary *)constantsToExport
 }
 
 #pragma mark - fs.stat
-RCT_EXPORT_METHOD(stat:(NSString *)target callback:(RCTResponseSenderBlock) callback)
+RCT_EXPORT_METHOD(stat:(NSString *)target verboseNull:(BOOL)verboseNull callback:(RCTResponseSenderBlock) callback)
```

Comment:
Does this just work as expected without adding an actual implementation of the added parameter?

Reply:
Ah, this was only the quick hotfix?

Reply:
Yeah, quick Saturday hotfix. I don't have context on what the intention behind this is in the first place, so left the empty implementation as a todo for the raptors that are working on ti

Reply:
@bryanvanwijk can you still make sure this gets properly fixed?

Reply:
The goal of this change is to gather insights short term so this change will do for now, we will ditch the code this or next release.

Reply:
Thanks @jaspermeijaard